app/models/reports/schema_v2.rb

```diff
@@ -176,15 +118,33 @@ def registered_patients_query_v2(region)
       end
     end
 
-    private
+    memoize def missed_visits(with_ltfu: false)
+      field = with_ltfu ? :missed_visit_lost_to_follow_up : :missed_visit_under_care
+      regions.each_with_object({}) { |region, hsh| hsh[region.slug] = sum(region, field) }
+    end
+
+    def missed_visits_rates(with_ltfu: false)
+      region_period_cached_query(__method__, with_ltfu: with_ltfu) do |entry|
+        slug, period = entry.slug, entry.period
+        numerator = missed_visits(with_ltfu: with_ltfu)[slug][period]
+        total = denominator(entry.region, period, with_ltfu: with_ltfu)
+        percentage(numerator, total)
+      end
+    end
+
+    alias_method :missed_visits_without_ltfu, :missed_visits
+    alias_method :missed_visits_without_ltfu_rates, :missed_visits_rates
 
-    # Return the actual 'active range' for a Region - this will be the from the first recorded at in a region until
-    # the end of the period range requested.
-    def active_range(region)
-      start = [earliest_patient_recorded_at_period[region.slug], periods.begin].compact.max
-      (start..periods.end)
+    def missed_visits_with_ltfu
+      missed_visits(with_ltfu: true)
     end
 
+    def missed_visits_with_ltfu_rates
+      missed_visits_rates(with_ltfu: true)
+    end
```

Comment:
Should we do this for the `_without_` versions as well, and remove the defaults? I feel that would be simpler because it would be more consistent / symmetric. Not a blocker though.

Reply:
I'm debating going to an API that actually just has the indicator name with the `with_ltfu` argument, and dropping the `with` and `with_out` suffixes entirely.  My plan is to have a rename / refactor PR that lands after v2 is the only code path, as it will be easier to clean things up w/o supporting both versions.